scripts/oracle_javadoc_seed.py

```
@@ -0,0 +1,195 @@
+
+
+def fetch_text(url: str) -> str:
+    print(f"fetch: {url}", file=sys.stderr)
+    request = urllib.request.Request(
+        url,
+        headers={
+            "User-Agent": "java-chat-doc-fetcher/1.0 (+https://localhost)",
+            "Accept": "text/plain,text/html,application/javascript,*/*;q=0.8",
+        },
+        method="GET",
+    )
```

Ruff (0.14.14)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)